<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>zNavigo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icones/web.png</normaloff>icones/web.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="vLayoutFenetre">
      <item>
       <layout class="QHBoxLayout" name="hLayoutBarreOutils">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="btPrecedent">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string comment="Page précédente" extracomment="Page précédente">Page précédente</string>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icones/prec.png</normaloff>icones/prec.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRepeatDelay">
           <number>0</number>
          </property>
          <property name="autoRepeatInterval">
           <number>0</number>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSuivant">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icones/suiv.png</normaloff>icones/suiv.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btActualiser">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icones/actu.png</normaloff>icones/actu.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btStopChargement">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icones/stop.png</normaloff>icones/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btHome">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icones/home.png</normaloff>icones/home.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="barreURL">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btAllerURL">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icones/go.png</normaloff>icones/go.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayoutTab">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabNavigateur">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>41</green>
                 <blue>41</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>41</green>
                 <blue>41</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>41</green>
                 <blue>41</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>41</green>
                 <blue>41</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <attribute name="title">
            <string>Nouvel onglet</string>
           </attribute>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>18</width>
              <height>28</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="vLayoutWeb">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QWebEngineView" name="web" native="true">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>138</red>
                     <green>226</green>
                     <blue>52</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>138</red>
                     <green>226</green>
                     <blue>52</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>138</red>
                     <green>226</green>
                     <blue>52</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>138</red>
                     <green>226</green>
                     <blue>52</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrirOnglet"/>
    <addaction name="actionFermerOnglet"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionPageprecedente"/>
    <addaction name="actionPageSuivante"/>
    <addaction name="actionActualiser"/>
    <addaction name="actionArreter"/>
    <addaction name="actionAccueil"/>
    <addaction name="actionAllerURL"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAProposDezNavigo"/>
    <addaction name="actionAProposDeQt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionOuvrirOnglet">
   <property name="text">
    <string>Ouvrir onglet</string>
   </property>
  </action>
  <action name="actionFermerOnglet">
   <property name="text">
    <string>Fermer onglet</string>
   </property>
  </action>
  <action name="actionPageprecedente">
   <property name="text">
    <string>Page précédente</string>
   </property>
  </action>
  <action name="actionPageSuivante">
   <property name="text">
    <string>Page suivante</string>
   </property>
  </action>
  <action name="actionAccueil">
   <property name="text">
    <string>Accueil</string>
   </property>
  </action>
  <action name="actionArreter">
   <property name="text">
    <string>Arrêter</string>
   </property>
  </action>
  <action name="actionAllerURL">
   <property name="text">
    <string>Aller à l'URL</string>
   </property>
  </action>
  <action name="actionActualiser">
   <property name="text">
    <string>Actualiser</string>
   </property>
  </action>
  <action name="actionAProposDezNavigo">
   <property name="text">
    <string>A propos de zNavigo</string>
   </property>
  </action>
  <action name="actionAProposDeQt">
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
